tests/test_client.py

```diff
@@ -143,3 +153,42 @@ def test_bad_endpoints(self):
                     "account": "",
                 },
             )
+
+
+@patch("pasqal_cloud.client.Auth0TokenProvider", FakeAuth0AuthenticationSuccess)
+class TestSDKAuth0AndEndpointsDicts(TestSDKCommonAttributes):
+    @pytest.mark.parametrize(
+        "env, core_endpoint_expected",
+        [
+            ("prod", "https://apis.pasqal.cloud/core-fast"),
+            ("preprod", "https://apis.preprod.pasqal.cloud/core-fast"),
+            ("dev", "https://apis.dev.pasqal.cloud/core-fast"),
+        ],
+    )
+    def test_existing_dict_keys(self, env, core_endpoint_expected):
```

Comment:
naming suggestion: `test_select_env` 

The name of the test (and the test itself) should be focused on the feature it tests instead of implementation details